docs/projects/amt/reporting-standard/schemas/systemcard_schema.json

```diff
@@ -93,6 +94,85 @@
             "type": "string",
             "description": "A short description of the system"
             },
+        "ai_act_profile": {
+            "type": "object",
+            "properties":{
+                "type":{
+                    "type": "string",
+                    "description": "System type",
+                    "enum": [
+                        "AI-systeem",
+                        "AI-systeem voor algemene doeleinden",
+                        "AI-model voor algemene doeleinden",
+                        "algoritme",
+                        "geen algoritme"
+                    ]
+                },
+                "open_source":{
+                    "type": "string",
+                    "description": "Whether the system is open source or not",
+                    "enum": [
+                        "open-source",
+                        "geen open-source"
+                    ]
+                },
+                "publication_category":{
+                    "type": "string",
+                    "description": "Publication category of the algorithm",
+                    "enum": [
+                        "impactvol algoritme",
+                        "niet-impactvol algoritme",
+                        "hoog-risico AI",
+                        "geen hoog-risico AI",
+                        "verboden AI",
+                        "uitzonderingsgrond: AI-verordening niet van toepassing"
+                    ]
+                },
+                "systemic_risk":{
+                    "type": "string",
+                    "description": "Whether the AI model is classified as having a systemic risk",
+                    "enum": [
+                        "systeemrisico",
+                        "geen systeemrisico"
+                    ]
+                },
+                "transparency_obligations":{
+                    "type": "string",
+                    "description": "Whether the system faces transparency obligations.",
+                    "enum": [
+                        "transparantieverplichtingen",
+                        "geen transparantieverplichtingen"
+                    ]
+                },
+                "role":{
+                    "type": "string",
+                    "description": "The organizations role in relation to the system.",
```

Comment:
Perhaps it is more consistent to choose one type of naming (instead of AI-model, algorithm or system).